### DISCUSSION TITLE
[Feature] Find recently added assets

### DISCUSSION THREAD

**Discussion by LaurentAtJoes** [immich-app/immich#9096] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

From what I understood in a discussion on discord, this feature existed but was removed.

I tried to find if there was already such a request, didn't find any, but apologies if there is one.

I mostly use external libraries (if that matters), and I need a way to find what was just scanned.

I regularly add batches of pictures spanning various periods, places, people...
and I need to easily browse through them, to check if all (relevant) people were detected, or maybe I want to add them to albums etc.

Any chance something could be added back ?
It could be a "recently added" section, or a way to search by scan date, or even just _sort_ by it...

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
I would love the ability to view the timeline by something other than the date that the image was taken. I know the way we load the timeline makes some things, like jumping to a specific date, challenging, but I think changing the sort order should be doable. I am hoping for the ability to view images sorted by the following SQL queries:

```
SELECT * FROM "assets" ORDER BY "createdAt" DESC;
SELECT * FROM "assets" ORDER BY "updatedAt" DESC;
```

**Comment by bo0tzz** [comment]
#8716
